customer-terminate-service/app/src/main/java/se/inera/intyg/cts/application/service/TerminationServiceImpl.java

```diff
@@ -79,20 +75,17 @@ public List<TerminationDTO> findAll() {
 
   /**
    * Resend password if all criteria are fulfilled
-   * @param terminationId
-   * @return
-   * @throws NotFoundException
+   * @param terminationId Id of the termination to update.
```

Comment:
Should maybe mention that it is the Id of the termination to resend the password for?